lang/lang/statics/base.stx

```diff
@@ -0,0 +1,359 @@
+module statics/base
+
+imports
+
+  signatures/constructors-sig
+  signatures/lex/literal-sig
+  signatures/pie-sig
+  statics/util
+  statics/type
+
+signature
+  sorts
+    ModuleTree
+    MODULE
+
+  // todo: move these to syntax/constructors.sdf3.
```

Comment:
There's a TODO here, and a couple of more TODOs in this PR. Do you intend to resolve these in a future PR? If not, please resolve them now or remove them.

Reply:
I'll not be resolving them now, so I'll remove them and make some issues I guess.

Reply:
Issues created, todos removed.
There is one remaining in `trans/gen/util.str`, that one is currently blocked until Spoofax 2.5.16 is released with a required Statix api.